mods/cnc/maps/scb30ea/scb30ea-AI.lua

```diff
@@ -0,0 +1,288 @@
+
+
+GDInuke1 = {type = "nuk2", pos = CPos.New(6,3), blocked = false}
```

Comment:
Our code style is to have a space after and before braces.
```suggestion
GDInuke1 = { type = "nuk2", pos = CPos.New(6,3), blocked = false }
```